NOTICE

```diff
@@ -1,3 +1,4 @@
 Kubernetes Anywhere
 Copyright 2015-2016 Weaveworks Ltd.
```

Comment:
Yes, per CLA we are. The intention was to cover for the dates prior to repo move... (cc @pidster)